### ISSUE TITLE
Goonstation

### ISSUE THREAD

**Issue by DavidEdwards** [yogstation13/yogstation-classic#495]
Continuation of the Goonstation issue, as I posted on the forum thread:

Our stance has now changed in regards to Goonstation code. We can now implement code from Goonstation.

Goonstation have permanently released their code:

https://github.com/goonstation/goonstation-2016

There is an issue on their licensing, which you can follow as guidelines:

https://github.com/goonstation/goonstation-2016/issues/5

Basically it comes down to this sentence:
**You can however reimplement things as long as you give attribution**

I will close this issue again after a week or so.

**Comment by Greatbigben** [comment]
Closing because this was 20 days ago
